PhotoBooth/CaptureSession.swift

```diff
@@ -0,0 +1,146 @@
+import UIKit
+import AVFoundation
+
+protocol PhotoCaptureable {
+    
+    func getPhotos() -> UIImage
+    
+    func configurePreview(view: UIView)
+}
+
+class CaptureSession: NSObject, PhotoCaptureable, AVCapturePhotoCaptureDelegate {
+    
+    var avSession = AVCaptureSession()
+    
+    var backCamera: AVCaptureDevice?
+    var frontCamera: AVCaptureDevice?
+    var currentCamera: AVCaptureDevice?
+    
+    var photoOutput: AVCapturePhotoOutput?
+    var videoOutput: AVCaptureVideoDataOutput?
+    let photoSessionPreset = AVCaptureSession.Preset.photo
+    let videoSessionPreset = AVCaptureSession.Preset.medium
+    
+    var cameraPreviewLayer: AVCaptureVideoPreviewLayer?
+    var cameraView = CameraView()
+    
+    var image: UIImage?
```

Comment:
what is this for?

Reply:
this is for the live preview of the camera